src/yamlist/args.rs

```diff
@@ -0,0 +1,80 @@
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+pub struct Args {
+    #[structopt(subcommand)]
+    pub mode: Mode,
+    
+    #[structopt(long, short)]
+    pub label: Option<String>,
+
+    #[structopt(long, short)]
+    pub out: Option<String>,
+}
+
+#[derive(StructOpt)]
+pub enum Mode {
+    #[structopt(about = "Convert from motion_list to yaml")]
+    Disasm {
+        file: String,
+    
+        #[structopt(long, short)]
+        label: Option<String>,
```

Comment:
According to the [docs page](https://docs.rs/structopt/0.3.3/structopt/#subcommands), "structopt also provides support for applications where certain flags need to apply to all subcommands, as well as nested subcommands". However, their example doesn't appear to demonstrate this. Can we implement this for this file?